src/components/step/HtmlStep/HtmlStep.stories.tsx

```diff
@@ -51,10 +51,10 @@ export const HtmlStepElementStyles: StoryObj<typeof HtmlStep> = {
 export const HtmlStepDefaultClassStyles: StoryObj<typeof HtmlStep> = {
     args: {
         html: `
-<div class="mdhui-step-title">title</div>
-<div class="mdhui-step-text">text</div>
-<button class="mdhui-step-next-button">next</button>
-<button class="mdhui-step-next-button" disabled>disabled next</button>
+<div class="title">title</div>
+<p>text</p>
+<button>Next</button>
+<button disabled>Disabled Next</button>
 <div class="mdhui-step-image">
```

Comment:
We can remove the mdhui-step-image stuff from stories, as it's not styled anymore